lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -859,12 +854,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
             opts = FOSS_DEFAULT_DOWNLOAD_URLS.merge(opts)
 
             case variant
-            when /^(fedora|el|centos)$/
+            when /^(fedora|el|centos|sles)$/
```

Comment:
we do have times when we're using this against `host['platform']` in general, however, so I'll fix those